interface/patient_file/transaction/print_referral.php

```diff
@@ -28,6 +28,7 @@
   'label_webpage_title'         => xlt('Referral Form'),
   'label_form1_title'           => xlt('Referral Form'),
   'label_name'                  => xlt('Name'),
+  'label_dob'                   => xlt('DOB'),
```

Comment:
Recommend adding locale formatting for DOB to match your locate date format setting in Config.
To do use the`oeFormatShortDate` function in the foreach/conditional around L-195.
E.g.
```
foreach ($patdata as $key => $value) {
    if ($key == "DOB") {
        $s = str_replace("{pt_$key}", text(oeFormatShortDate($value)), $s);
    }
    if ($key == "sex") {
        $s = str_replace("{pt_$key}", generate_display_field(array('data_type' => '1','list_id' => 'sex'), $value), $s);
    } else {
        $s = str_replace("{pt_$key}", text($value), $s);
    }
}
````

Reply:
Thanks!